src/sonic-yang-models/yang-models/sonic-portchannel.yang

```diff
@@ -95,7 +95,21 @@ module sonic-portchannel {
                     mandatory true;
                     type stypes:admin_status;
                 }
-            } /* end of list PORTCHANNEL_LIST */
+
+                leaf lacp_key {
```

Comment:
Kindly add test cases in https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-yang-models/tests/yang_model_tests/tests_config/portchannel.json.

And Sample config in https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-yang-models/tests/files/sample_config_db.json. Thx